### ISSUE TITLE
After 3.3 upgrade, only "Welcome to nginx!" is shown

### ISSUE THREAD

**Issue by strickes** [zammad/zammad-docker-compose#147]
### Infos:

* Docker version: Docker version 19.03.5, build 633a0ea838
* Docker-compose version: docker-compose version 1.25.3, build d4d1b42b
* Operating system (Docker host): Ubuntu 18.04 (Linux web06 4.15.0-62-generic #69-Ubuntu SMP Wed Sep 4 20:55:53 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux)

### Expected behavior:

Zammad is shown after a docker-compose pull upgrade from 3.2 to 3.3.

### Actual behavior:

only the default nginx page ("Welcome to nginx!") is shown when using the latest docker-compose. I upgraded from the previous versions via docker-compose pull.

### Steps to reproduce the behavior:

*

**Comment by sscholl** [comment]
+1
https://github.com/zammad/zammad-docker-compose/commit/f93db7f2e8e7183266be95fdeb6ec6a2bee09d26

This was deleted:
```
  # configure nginx	
  if ! env | grep -q KUBERNETES; then	
    sed -e "s#server .*:3000#server ${ZAMMAD_RAILSSERVER_HOST}:${ZAMMAD_RAILSSERVER_PORT}#g" -e "s#server .*:6042#server ${ZAMMAD_WEBSOCKET_HOST}:${ZAMMAD_WEBSOCKET_PORT}#g" -e "s#server_name .*#server_name ${NGINX_SERVER_NAME};#g" -e 's#/var/log/nginx/zammad.\(access\|error\).log#/dev/stdout#g' < contrib/nginx/zammad.conf > /etc/nginx/sites-enabled/default	
  fi
```

**Comment by monotek** [comment]
Whoops! Only the if clause was meant to be deleted.
Will be fixed with: #148

But how do you've ran into this?
The .env file was pinned to version tag 3.3.0-11 which still has the right config inside.
Did you used "latest" tag in the env file?

**Comment by strickes** [comment]
> Whoops! Only the if clause was meant to be deleted.
> Will be fixed with: #148
> 
> But how do you've ran into this?
> The .env file was pinned to version tag 3.3.0-11 which still has the right config inside.
> Did you used "latest" tag in the env file?

yes I actually did use the latest tag :-( guess that was my fault. Thanks for fixing it so fast.
